firefox-ios/Client.xcodeproj/project.pbxproj

```diff
@@ -10998,6 +11006,10 @@
 				AB2AC6652BD15E6300022AAB /* CertificatesHandler.swift */,
 				AB936A682C05F2B100600F82 /* TrackingProtectionButton.swift */,
 				0A4978492C53E63200B1E82A /* TrackingProtectionViewController.swift */,
+				AB9A0C002C6CBC9100BFA22A /* TrackingProtectionDetailsViewController.swift */,
+				ABE856AA2C74F23200C56F47 /* TrackingProtectionHeaderView.swift */,
+				ABE856AC2C75029F00C56F47 /* TrackingProtectionStatusView.swift */,
+				AB8EE2D22C750E6100BD0A6B /* TrackingProtectionVerifiedByView.swift */,
```

Comment:
nit: if theres are all related to detail view, group own folder?

Reply:
had some issues with folders when switching branches or applying stashes so I preferred to avoid this step until it's safer to do so

Reply:
np!